### PR TITLE
Sms notification endpoint

### PR DESCRIPTION
**Pull request by moseti1**
This is PR to add sms notification ability to the app. Requires future enhancement.

### CODE REVIEW

File: app/database.py [hunk 1]
```diff
@@ -11,7 +12,7 @@ def __init__(cls, db_url):
     @classmethod
     def init_db(cls, db_url):
         try:
-            cls.connection = psycopg2.connect(db_url)
```
Comment (Dave-mash) [inline app/database.py]: This line is perfectly valid

File: requirements.txt [hunk 2]
```diff
@@ -1,44 +1,59 @@
-astroid==2.2.0
-atomicwrites==1.3.0
+appdirs==1.4.3
```
Comment (Dave-mash) [inline requirements.txt]: Please don't remove dependencies.

File: requirements.txt [hunk 3]
```diff
@@ -1,44 +1,59 @@
...
 mccabe==0.6.1
 more-itertools==6.0.0
+msgpack==0.6.2
+packaging==20.3
+pep517==0.8.2
 pep8==1.7.1
 phonenumbers==8.12.9
 pluggy==0.9.0
-psycopg2==2.7.7
-psycopg2-binary==2.7.7
```
Comment (Dave-mash) [inline requirements.txt]: Don't remove dependencies

File: app/database.py [hunk 4]
```diff
@@ -11,7 +12,7 @@ def __init__(cls, db_url):
...
+            cls.connection = psycopg2.connect(user="postgres",password="postgres",host="127.0.0.1",port="5432",database='zaobora_database')
```
Comment (Dave-mash) [inline app/database.py]: Revert back to line 14

**Review by Dave-mash — CHANGES_REQUESTED** [review]
Please avoid modifying code that is already working as it'll break for everyone else.

**Review by moseti1 — COMMENTED** [review]
modified the  affected items in the previous commit.

File: requirements.txt [hunk 5]
```diff
@@ -1,44 +1,60 @@
...
+retrying==1.3.3
 six==1.12.0
+SQLAlchemy==1.3.19
 toml==0.10.1
-typed-ast==1.2.0
 urllib3==1.25.10
+webencodings==0.5.1
 Werkzeug==0.14.1
 wrapt==1.11.1
+
```
Comment (Dave-mash) [inline requirements.txt:51]: I can see you've added extra dependencies like postgres. Is your code still breaking?

File: app/database.py [hunk 6]
```diff
@@ -11,7 +11,7 @@ def __init__(cls, db_url):
...
```
Comment (Dave-mash) [inline app/database.py]: Please if your code is breaking I can help you resolve it. Changing lines like this is jot ideal

**Review by Dave-mash — COMMENTED** [review]
The .env.example is important you shouldn't have deleted it.

File: requirements.txt (hunk 5 above)
Comment (moseti1) [reply requirements.txt:51]: nope working fine right now. Those are the dependencies i have in my local repository.Working fine.

File: app/database.py (hunk 6 above)
Comment (moseti1) [reply app/database.py]: Code working fine locally. Would like help better code commit practices.

**Review by moseti1 — COMMENTED** [review]
cross-checked the default branch dependencies ,database and pay modules.

**Review by Dave-mash — DISMISSED** [review]

**Review by Jordan-type — DISMISSED** [review]

**Comment by Dave-mash** [comment]
> This is PR to add sms notification ability to the app. Requires future enhancement.

looks like you have conflicts just restore the .env.example right here on github

**Pull request merged**